osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearMBusStatusOnAllChannelsCommandExecutorTest.java

```diff
@@ -83,122 +93,139 @@ class ClearMBusStatusOnAllChannelsCommandExecutorTest {
 
   @BeforeEach
   void setup() {
-    this.executor = new ClearMBusStatusOnAllChannelsCommandExecutor(this.objectConfigServiceHelper);
+    this.executor =
+        new ClearMBusStatusOnAllChannelsCommandExecutor(
+            this.objectConfigServiceHelper, this.adhocService);
     this.dlmsMessageListener = new LoggingDlmsMessageListener(null, null);
   }
 
   @Test
-  void testExecuteObjectNotFound() throws ProtocolAdapterException {
-    when(this.objectConfigServiceHelper.findDefaultAttributeAddress(any(), any(), any(), any()))
-        .thenThrow(new ProtocolAdapterException("Object not found"));
-
+  void testExecuteObjectNotFound() throws OsgpException {
```

Comment:
If you extend this test by returning two occupied channels on scanchannels and both throwing an exception you can test not skipping rest of channels when first throws exception and bundling both exceptions in response.